utils/opensc/Makefile

```diff
@@ -8,9 +8,9 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=opensc
-PKG_VERSION:=0.22.0
-PKG_RELEASE:=$(AUTORELEASE)
-PKG_HASH:=8d4e5347195ebea332be585df61dcc470331c26969e4b0447c851fb0844c7186
+PKG_VERSION:=0.24.0
```

Comment:
Indeed, this is the fastest approach, but looking at the number of changes:

```
$ git diff --stat 0.22.0..0.24.0 | grep changed
 501 files changed, 42206 insertions(+), 12300 deletions(-)
```

it seems to me like a big regression potential, so I'm wondering if it would be possible to identify just the security fixes and backport them as individual patches instead?

Reply:
the upgrade is not only for CVE but also for the capability to use pkcs11-tool to encrypt/decrypt with sympetric keys.

Reply:
> for the capability to use pkcs11-tool to encrypt/decrypt with sympetric keys.

Quoting https://github.com/openwrt/packages/blob/master/CONTRIBUTING.md#release-branches

* Please ONLY cherry-pick or commit security and bug-fixes to these branches.
* Do NOT add new packages and do NOT do major upgrades of packages here.

Reply:
so we need a solution on PRPL for adding new pages or doing major upgrade?
Will we fork these repo for PRPL to have more flexibility ?

Reply:
> so we need a solution on PRPL for adding new packages or doing major upgrade?

This is OpenWrt project, so that is off-topic here.

>  Will we fork these repo for PRPL to have more flexibility ?

An "upstream first" policy refers to the practice of contributing code changes to the original or main source code repository ("upstream") before using or applying these changes in your own project or a derivative ("downstream"). This approach ensures that improvements and fixes benefit the entire community and maintain compatibility with the main codebase. Forks, or separate branches of the codebase, should be avoided when possible as they can lead to fragmentation, making it harder to maintain software and integrate future updates from the original source.